grapher/Form1.cs

```diff
@@ -26,21 +28,30 @@ public RawAcceleration()
         {
             InitializeComponent();
 
-            ManagedAccel activeAccel = null;
+            Version assemVersion = typeof(RawAcceleration).Assembly.GetName().Version;
+            Version driverVersion = null;
 
             try
             {
-                activeAccel = ManagedAccel.GetActiveAccel();
+                driverVersion = VersionHelper.ValidateAndGetDriverVersion(assemVersion);
             }
-            catch (DriverNotInstalledException ex)
+            catch (VersionException ex)
             {
-                MessageBox.Show($"Driver not installed.\n\n {ex.ToString()}");
+                MessageBox.Show(ex.Message);
                 throw;
             }
+            
+            ToolStripMenuItem HelpMenuItem = new ToolStripMenuItem("&Help");
+
+            HelpMenuItem.DropDownItems.AddRange(new ToolStripItem[] {
+                    new ToolStripMenuItem("&About", null, (s, e) => new AboutBox(driverVersion).ShowDialog())
```

Comment:
Out of curiosity, what is the & for in "&About"?

Reply:
makes first char the shortcut key: https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.menuitem.shortcut?view=netframework-4.8

Reply:
Nice